sound/soc/codecs/rt711-sdca.c

```diff
@@ -642,6 +642,77 @@ static int rt711_sdca_set_gain_get(struct snd_kcontrol *kcontrol,
 	return 0;
 }
 
+static void rt711_sdca_set_fu0f_capture_ctl(struct rt711_sdca_priv *rt711)
+{
+	int mute = rt711->fu0f_dapm_mute || rt711->fu0f_mixer_mute;
```

Comment:
could make it `bool` but even better I'd just do `if (rt711->fu0f_dapm_mute || rt711->fu0f_mixer_mute)` below

Reply:
> could make it `bool` but even better I'd just do `if (rt711->fu0f_dapm_mute || rt711->fu0f_mixer_mute)` below

Updated. Thanks.